exercises/F-strings-methods/exercise2.js

```diff
@@ -1,3 +1,4 @@
 const nameD = " Daniel  ";
 let message ="My name is " +nameD.trim() +" and my name is " + nameD.trim().length +" characters long.";
+//print the message
```

Comment:
Hi as a new developer it is better to use let or const instead of using var. The fact about const and let is you can’t re-assign another value to the constant x, however let allows you to declare variables.